pystages/vector.py

```diff
@@ -148,6 +148,24 @@ def __eq__(self, other):
                 return False
         return True
 
+    def __mul__(self, other):
```

Comment:
Current implementation modifies `self`, it is better to return a new instance of `Vector` and keep `self` unmodified.

If user wants the currently implemented behavior, he should write `v *= scalar`.